GraalVM/CE/Dockerfile.ol8

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2020 Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/
+
+FROM oraclelinux:8-slim
+
+# Note: If you are behind a web proxy, set the build variables for the build:
+#       E.g.:  docker build --build-arg "https_proxy=..." --build-arg "http_proxy=..." --build-arg "no_proxy=..." ...
+
+ARG GRAALVM_VERSION=20.0.0
+ARG JAVA_VERSION=java8
+ARG GRAALVM_PKG=https://github.com/graalvm/graalvm-ce-builds/releases/download/vm-$GRAALVM_VERSION/graalvm-ce-$JAVA_VERSION-linux-amd64-$GRAALVM_VERSION.tar.gz 
+ENV LANG=en_US.UTF-8 JAVA_HOME=/opt/graalvm-ce-$JAVA_VERSION-$GRAALVM_VERSION/
+
+RUN microdnf update -y oraclelinux-release-el8 \
+    && microdnf install -y bzip2-devel ed gcc gcc-c++ gcc-gfortran gzip file fontconfig less libcurl-devel make openssl openssl-devel readline-devel tar vi which xz-devel zlib-devel \
```

Comment:
Please could you also add the `findutils` package, the gu-wrapper requires the `find` utility.

Reply:
`microdnf` also doesn't need the `-y` parameter, but it's not a showstopper. If you're editing the file anyway, you may as well remove them though.

Reply:
@mzachh @Djelibeybi Both changes are committed

Reply:
What's the reason for having two calls to `microdnf`? Why not just install everything at once, i.e. what am I missing here? :)

Reply:
You're right! I made the requested changes